src/joinRates/columns.tsx

```diff
@@ -0,0 +1,14 @@
+export const getJoinRatesColumns = (columns: Array<string>) =>
+  columns.map(column => {
+    return {
+      id: column,
+      title: column,
+      dataIndex: column,
+      render: (data: any) => {
+        if (column === 'table_names') {
```

Comment:
BTW, here we have column 'table_names'. But in the data, we have 'Table names'.
It means that you will not see the first column with table names.